pkg/middleware/recovery/recovery_test.go

```diff
@@ -0,0 +1,210 @@
+package recovery
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials/insecure"
+	"google.golang.org/grpc/status"
+	"google.golang.org/grpc/test/bufconn"
+
+	openfgav1 "github.com/openfga/api/proto/openfga/v1"
+
+	"github.com/openfga/openfga/pkg/logger"
+)
+
+func TestPanic(t *testing.T) {
+	logger := logger.MustNewLogger("text", "info", "unix")
+	handlerFunc := http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		panic("Unexpected error!")
+	})
+
+	t.Run("With panic Recovery", func(t *testing.T) {
```

Comment:
For searchability of tests, we have established a convention of using underscores in our test names instead of whitespace. This is because Go injects underscores in place of whitespace anyways.

So please rename this and others using underscores. For example, `with_panic_recovery`.

Reply:
Done